patch/preconfig/packaging-update-abiname-to-11-2.patch

```diff
@@ -0,0 +1,20 @@
+Update debian abiname to 9-2
+
+From: Guohan Lu <gulv@microsoft.com>
+
+
```

Comment:
Please also remove old packaging-update-abiname-to-9-2.patch, or rename it by git.

Reply:
Done